### ISSUE TITLE
New Feature Request

### ISSUE THREAD

**Issue by AbbyKimball** [JinmiaoChenLab/cytofkit#14]
It would be incredibly helpful to be able to export events from a particular cluster in order to analyze them in FlowJo, is there a way to do this, or any hope of this becoming a feature in newer versions? Thanks!

**Comment by MattMyint** [comment]
Currently, the `cytof_clusterMtrx` function generates a matrix of expression values for cells in a chosen cluster, using analysis results of the main cytofkit function as its input.

**Comment by sjm0240111** [comment]
you want to send the data back to flowJo right? I'm also wondering how to do this. Have you figured out the solution? @AbbyKimball 
by the way, this package is extremely useful for me, and it's so user friendly!

**Comment by AbbyKimball** [comment]
@sjm0240111 yep, I would love to export individual clusters so I could see them in FlowJo, but I've found no way to do it. Right now I've been just taking clusters with certain combinations of markers and trying to find them in FlowJo. Here's an example of what I've done.
[EC20170919 finding populations by flowjo.pdf](https://github.com/JinmiaoChenLab/cytofkit/files/1479328/EC20170919.finding.populations.by.flowjo.pdf)

**Comment by MattMyint** [comment]
Oh, seems I misunderstood the feature request. Apologies, the `cytof_clusterMtrx()` function only exports a csv file.

We'll work on implementing a function to save new fcs files with only events from a specific cluster. That's what you meant right?

**Comment by AbbyKimball** [comment]
Yep! Thanks!
